web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -196,6 +196,13 @@
     "db.numidle": "Available connections",
     "db.openattempt": "Status",
     "db.url": "JDBC URL",
+    "db.product": "DB Product",
```

Comment:
Oh also I just thought that the "DB" prefix on these labels is probably not needed as we are already in the "Database" section; could you remove them to reduce the visual clutter? Thanks!